docs/src/modules/components/AdInHouse.js

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable material-ui/no-hardcoded-labels, react/no-danger */
+import React from 'react';
+import propTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    display: 'block',
+    overflow: 'hidden',
+    backgroundColor: theme.palette.background.level2,
+    padding: `${theme.spacing(1.5)}px ${theme.spacing(1.5)}px ${theme.spacing(
+      1.5,
+    )}px ${theme.spacing(1.5) + 130}px`,
+    borderRadius: theme.shape.borderRadius,
+    '& $imageWrapper': {
+      float: 'left',
+      marginLeft: -130,
+      marginRight: theme.spacing(1.5),
+    },
+    '& img': {
+      verticalAlign: 'middle',
+    },
+    '& a, & a:hover': {
+      color: theme.palette.text.primary,
+      textDecoration: 'none',
+    },
+    '& $description': {
+      ...theme.typography.body2,
+      display: 'block',
+    },
+    '& $poweredby': {
+      ...theme.typography.caption,
+      color: theme.palette.text.secondary,
+      display: 'block',
+    },
+  },
+  imageWrapper: {},
+  description: {},
+  poweredby: {},
+}));
+
+export default function AdInHouse(props) {
+  const { ad } = props;
+  const classes = useStyles();
+  return (
+    <span className={classes.root} id="in-house">
+      <a
+        className={classes.link}
+        href={ad.link}
+        // eslint-disable-next-line react/jsx-no-target-blank
+        target="_blank"
+        rel="nofollow noopener"
```

Comment:
Can you refresh my mind why we do this? It's not user generated content so the spam rationale doesn't apply, no?

Otherwise reading https://support.google.com/webmasters/answer/96569?hl=en we maybe want to use `sponsored` instead. Otherwise I don't understand why we wouldn't want to be associated with that site if we authored the link.

[`sponsored` announcement](https://webmasters.googleblog.com/2019/09/evolving-nofollow-new-ways-to-identify.html)

Reply:
@eps1lon Thanks for linking this SEO news. It has been a long time since Google changes it.

Carbon and CodeFund use "noopener" for security reasons. Carbon should have been using "nofollow" (only CodeFund did). Given

> Google also required nofollow for paid or sponsored links. If you were caught accepting anything of value in exchange for linking out without the nofollow attribute, Google could penalize you.

https://moz.com/blog/nofollow-sponsored-ugc. I will notify Carbon and CodeFund so they start to use the `sponsored` value.

I also think that we should do a follow-up PR to use sponsored where appropriate. I have added it in the list of small changes to batch, that I can handle unless somebody wants to take care of it. I have noticed that Vue already uses ` rel="sponsored noopener"`.

Reply:
When things go wrong 🙃: https://www.brinzan.com/2019/09/apache-software-foundation-selling-links-for-seo/

Reply:
Oh yeah the noopener I do understand. I just don't understand the nofollow for links that obviously relate to our content. As I understood the [google guidelines](https://support.google.com/webmasters/answer/35769?hl=en) simply getting money from a linked page doesn't imply a link scheme. Only if we solely link because we got money. And as far as I can tell the linked content clearly relates to ours. Same for github or material.io. This is why I wanted this clarification.

For links to sponsors I completely understand using nofollow or better sponsored.

Reply:
Do I bring in extra information by saying that follow links have more weight than nofollow links in the search engine results pages (SERP) position?